### PR TITLE
Fix linter issues

### PR DESCRIPTION
**Pull request by alexandrevicenzi**
- Fixed code styling
- Fixed most pylint issues

### CODE REVIEW

File: ubipop/__init__.py [hunk 1]
```diff
@@ -163,7 +169,7 @@ def _get_repo_counterpart(self, input_repo, repos_to_match):
             if input_repo.arch == repo.arch and \
                     input_repo.platform_full_version == repo.platform_full_version:
                 return repo
-
+        return None
```
Comment (danrodrig) [inline ubipop/__init__.py]: Notice that we are no longer enforcing inconsistent-return-statements

Comment (rbikar) [reply ubipop/__init__.py]: This also a reason for failing test coverage check.

**Comment by alexandrevicenzi** [comment]
@rbikar I try to squash to not pollute the history, but good point about review.

**Comment by rbikar** [comment]
> @rbikar I try to squash to not pollute the history, but good point about review.

I always squash before merging to master (there is an option for it: squash and merge). So history is not polluted.

**Pull request merged**